draft-tulshibagwale-oauth-transaction-tokens-02.md

```diff
@@ -487,29 +506,39 @@ A workload within a call chain may request the Transaction Token Server to repla
 
-### Transaction Token Server Responsibilities
-A Transaction Token Server replacing a Txn-Token must consider that modifying previously asserted values from existing Txn-Tokens can completely negate the benefits of Txn-Tokens. When issuing replacement Txn-Tokens, a Transaction Token Server therefore:
+### Txn-Token Service Responsibilities
+A Txn-Token Service replacing a Txn-Token must consider that modifying previously asserted values from existing Txn-Tokens can completely negate the benefits of Txn-Tokens. When issuing replacement Txn-Tokens, a Transaction Token Server therefore:
 
 * MAY enable modifications to asserted values that reduce the scope of permitted actions
 * MAY enable reduction of token bloat by removing nesting, and placing workload identifiers as asserted values instead
 * MAY enable additional asserted values 
 * SHOULD NOT enable modification to asserted values that expand the scope of permitted actions
 
 ### Replacement Txn-Token Request
-To request a replacement Txn-Token, the requester makes a Txn-Token Request {{txn-token-request}} but includes the Txn-Token to be replaced as the value of the `subject_token` parameter.
+To request a replacement Txn-Token, the requester makes a Txn-Token Request as described in {{txn-token-request}} but includes the Txn-Token to be replaced as the value of the `subject_token` parameter.
 
 ### Replacement Txn-Token Response
-A successful response by the Transaction Token Server to a Replacement Txn-Token Request is a Txn-Token Response {{txn-token-response}}
+A successful response by the Transaction Token Server to a Replacement Txn-Token Request is a Txn-Token Response as described in {{txn-token-response}}
 
 ### Removing Nesting
 A Replacement Txn-Token Request MAY include a Nested Txn-Token in its request. If the request is successful, the Transaction Token Server SHALL always respond with a Leaf Txn-Token. 
 
 If the Replacement Txn-Token Request has a Nested Txn-Token in the request's `subject_token` parameter, then the Transaction Token Server MAY include information about services that had signed the Nested Txn-Token that is requested to be replaced.
 
-If the Transaction Token Server wishes to include information about any nested Txn-Token signers, then it SHALL include a field named `previous_signers` in the `azc` value of the Txn-Token that it issues. The value of this field MUST be an array of strings. Each string is the the value of the `iss` field of a Nested Txn-Tokens received in the Replacement Txn-Token Request. Note that
+If the Transaction Token Server wishes to include information about any nested Txn-Token signers, then it SHALL include a field named `previous_signers` in the `azc` value of the Txn-Token that it issues. The value of this field MUST be an array of strings. Each string is the the value of the `iss` field of a Nested Txn-Tokens received in the Replacement Txn-Token Request. Note that:
```

Comment:
Are there any recommendations on the type of data to include or not include in `azc`? Or is it up to the individuals to encrypt if there is any sensitive data in it?